docs/grpc_examples.md

```diff
@@ -0,0 +1,19 @@
+# Mnist gRPC Example
+
+To run the Keras Mnist gRPC example run:
+
+```bash
+python -m colearn_examples.grpc.run_grpc_demo.py --n_learners 5 --dataloader_tag KERAS_MNIST --model_tag KERAS_MNIST \
```

Comment:
I think this should be colearn_examples.grpc.run_grpc_demo without the .py on the end?